### PR TITLE
New effect: Spawn Roasting Lamar

### PR DESCRIPTION
**Pull request by juhana**
Spawns Lamar who greets the player. The player character dismisses him and Lamar starts insulting the player. When the effect ends Lamar flips off the player and leaves. The player character expresses confusion.

...it's the meme.

### CODE REVIEW

**Comment by TheRealAyCe** [comment]
Could maybe even replace Lamar with a random ped model.

File: ChaosMod/Effects/db/Peds/PedsSpawnRoastingLamar.cpp [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+	Hash relationshipGroup;
+	ADD_RELATIONSHIP_GROUP("_ROASTING_LAMAR", &relationshipGroup);
+	SET_RELATIONSHIP_BETWEEN_GROUPS(0, relationshipGroup, GET_HASH_KEY("PLAYER"));
+	SET_RELATIONSHIP_BETWEEN_GROUPS(0, GET_HASH_KEY("PLAYER"), relationshipGroup);
+
+	Ped playerPed = PLAYER_PED_ID();
+	Vector3 playerPos = GET_ENTITY_COORDS(playerPed, false);
+
+	lamarPed = CreatePoolPed(4, model, playerPos.x, playerPos.y, playerPos.z, GET_ENTITY_HEADING(playerPed));
```
Comment (pongo1231) [inline ChaosMod/Effects/db/Peds/PedsSpawnRoastingLamar.cpp:21]: Might make more sense to use the normal `CREATE_PED` instead considering it gets cleaned up by the effect itself afterwards anyways.

**Comment by pongo1231** [comment]
Lamar should probably be invincible as well since you can still blow him up with explosives.

**Comment by juhana** [comment]
Lamar is now invincible during the roasting.

**Pull request merged**